.eslintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     'plugin:eslint-plugin/recommended',
     'prettier',
   ],
-  plugins: ['eslint-plugin', 'prettier'],
+  plugins: ['es5', 'eslint-plugin', 'prettier'],
```

Comment:
I would prefer https://github.com/mysticatea/eslint-plugin-node and [`no-unsupported-features`](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unsupported-features.md)

Reply:
I'll take a look at this and push up a new commit - thank you for the review. 🙂